cmd/bee/cmd/start_windows.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build windows
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+	"golang.org/x/sys/windows/svc/eventlog"
+
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/sirupsen/logrus"
+)
+
+func isWindowsService() (bool, error) {
+	return svc.IsWindowsService()
+}
+
+func createWindowsEventLogger(svcName string, logger logging.Logger) (logging.Logger, error) {
+	el, err := eventlog.Open(svcName)
+	if err != nil {
+		return nil, err
+	}
+
+	winlog := &windowsEventLogger{
+		logger: logger,
+		winlog: el,
+	}
+
+	return winlog, nil
+}
+
+type windowsEventLogger struct {
+	logger logging.Logger
+	winlog debug.Log
+}
+
+func (l *windowsEventLogger) Tracef(format string, args ...interface{}) {
+	// ignore
+}
+
+func (l *windowsEventLogger) Trace(args ...interface{}) {
+	// ignore
+}
+
+func (l *windowsEventLogger) Debugf(format string, args ...interface{}) {
+	// ignore
```

Comment:
does this mean we will never be able to log anything > INFO?

Reply:
Yes, as we talked about, 'Debug' and 'Trace' level logs will not be pushed to Windows Event Log.

While it could be possible to setup (which I already did and tested) to log those lower levels, I am not completely confident in Windows Event Log to push possibly huge amount of logs into. If we were actually developing for Windows, I assume we would have greater knowledge about this and could made assumptions what can and cannot be done without problems. As for now, I think better option is to just not log anything under `Info` level.

Reply:
:+1: